src/status_im/contexts/chat/messenger/composer/link_preview/events.cljs

```diff
@@ -82,22 +82,45 @@
 
 (rf/defn unfurl-parsed-urls-success
   {:events [:link-preview/unfurl-parsed-urls-success]}
-  [{:keys [db]} request-id {new-previews :linkPreviews}]
+  [{:keys [db]} request-id {new-previews :linkPreviews status-link-previews :statusLinkPreviews}]
   (when (= request-id (get-in db [:chat/link-previews :request-id]))
-    (let [new-previews         (map data-store.messages/<-link-preview-rpc new-previews)
-          curr-previews        (get-in db [:chat/link-previews :unfurled])
-          indexed-new-previews (utils.collection/index-by :url new-previews)]
+    (let [new-previews             (map data-store.messages/<-link-preview-rpc new-previews)
+          new-status-link-previews (map data-store.messages/<-status-link-previews-rpc
+                                        status-link-previews)
+          merged-preview-types     (-> (map
+                                        (fn [{{:keys [display-name members-count banner icon
+                                                      description]} :community
+                                              url                   :url}]
+                                          ;; Handle community type, Other implementations will have
```

Comment:
I couldn't understand this comment. What do you mean by overwrite? Which other implementations will have to overwrite?